lib/commands/dump.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+module.exports = {
+	/**
+	 * @param {Object} data
+	 * @param {Object} data.parameters Additional arguments provided by the user.
+	 * @param {String} data.packageName Name of current package to process.
+	 * @param {Options} data.options The options object.
+	 * @param {Repository|null} data.repository
+	 * @returns {Promise}
+	 */
+	execute( data ) {
+		const log = require( '../utils/log' )();
+		const execCommand = require( './exec' );
+
+		return new Promise( ( resolve, reject ) => {
+			execCommand.execute( getExecData( 'git rev-parse HEAD' ) )
```

Comment:
The first you know because this command is called of existing packages, the latter is simple. But ok.